modin/pandas/test/test_io.py

```diff
@@ -1115,6 +1115,22 @@ def get_internal_df(df):
         for name1, name2 in zip(get_internal_df(read_df).index.names, [None, "a"]):
             assert name1 == name2
 
+    def test_read_csv_empty_frame(
+        self,
+        request,
+    ):
+        if request.config.getoption("--simulate-cloud").lower() != "off":
```

Comment:
Can you consider this comment?
https://github.com/modin-project/modin/issues/3295#issuecomment-889047542

Reply:
Yes, fixed.